packages/dds/map/src/directory.ts

```diff
@@ -338,6 +338,8 @@ export class DirectoryFactory {
  * @sealed
  */
 export class SharedDirectory extends SharedObject<ISharedDirectoryEvents> implements ISharedDirectory {
+    private _dispose = false;
```

Comment:
Should handling of disposed be delegated fully to this.root? Otherwise it's half baked system - we set boolean on this object, but never check for it, which will be source of bugs...

Reply:
Can you please explain what  do you mean by it? How will the root handle the dispose?

Reply:
Deleted this from here and delegated to root.